alignak_module_backend/broker/module.py

```diff
@@ -861,7 +864,7 @@ def manage_brok(self, brok):
                 logger.debug("Got contact update status brok: %s", brok.data)
                 self.update_status(brok)
 
-            if brok.type == 'new_conf':
+            if brok.type == 'new_conf' or not self.loaded:
```

Comment:
Did you already missed the `new_conf` brok? If it should happen, then there is an important problem with Alignak and its configuration dispatcher ... then it would be preferable to raise an ERROR log, no ?